src/Zed/ruleset.xml

```diff
@@ -12,117 +12,99 @@
 
     <!-- BUSINESS FACADE RULES -->
     <rule
-        name="AllMethodsPublicInFacadeRule"
-        message="Zed Business Layer - Facade: {0}"
-        class="ArchitectureSniffer\Zed\Business\Facade\AllMethodsPublicInFacadeRule">
+            name="AllMethodsPublicInFacadeRule"
+            message="Zed Business Layer - Facade: {0}"
+            class="ArchitectureSniffer\Zed\Business\Facade\AllMethodsPublicInFacadeRule">
 
         <priority>1</priority>
     </rule>
 
     <rule
-        name="ArgumentsFacadeRule"
-        message="Zed Business Layer - Facade: {0}"
-        class="ArchitectureSniffer\Zed\Business\Facade\ArgumentsFacadeRule">
+            name="ArgumentsFacadeRule"
+            message="Zed Business Layer - Facade: {0}"
+            class="ArchitectureSniffer\Zed\Business\Facade\ArgumentsFacadeRule">
 
-        <priority>2</priority>
+        <priority>3</priority>
```

Comment:
```
We decided to use two priorities:
Prio 1: API-related (e.g. something in a facade) OR runtime-critical (e.g. "No call from Client to Zed")
Prio 2: Everyhing else
```
Should we mention somewhere that we have also 3? Is that for currently not fixable rules?